target/linux/ath79/dts/qca9558_comfast_cf-wr650ac.dtsi

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9557.dtsi"
+
+/ {
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wps {
+			label = "wr650ac:blue:wps";
```

Comment:
Sorry if i might have not been 100% precise - the LED definitions have to be put into the device-specific dts. This is due to the v1's boardname being `cf-wr650ac-v1` and the v2's `cf-wr650ac-v2`

Reply:
done